lib/main/FreeRTOS/Source/portable/GCC/ARM_CM4F/port.c

```diff
@@ -277,6 +277,7 @@ static void prvPortStartFirstTask( void )
 					" isb					\n"
 					" svc 0					\n" /* System call to start first task. */
 					" nop					\n"
+					" .ltorg                                \n"
```

Comment:
Local table is too far on gcc8, failing LTO link
https://bugs.launchpad.net/gcc-arm-embedded/+bug/1763050